content/ja/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -0,0 +1,144 @@
+---
+title: ネットワークポリシーを宣言する
+min-kubernetes-server-version: v1.8
+content_type: task
+---
+
+<!-- overview -->
+このドキュメントでは、Pod同士の通信を制御するネットワークポリシーを定義するための、Kubernetesの[NetworkPolicy API](/ja/docs/concepts/services-networking/network-policies/)を使い始める手助けをします。
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+ネットワークポリシーをサポートしているネットワークプロバイダーが設定済みであることを確認してください。さまざまなネットワークプロバイダーがNetworkPolicyをサポートしています。次に挙げるのは一例です。
+
+* [Calico](/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy/)
+* [Cilium](/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy/)
+* [Kube-router](/docs/tasks/administer-cluster/network-policy-provider/kube-router-network-policy/)
+* [Romana](/docs/tasks/administer-cluster/network-policy-provider/romana-network-policy/)
+* [Weave Net](/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy/)
+
+{{< note >}}
+上記のリストは製品名のアルファベット順にソートされていて、推奨順や好ましい順にソートされているわけではありません。このページの例は、Kubernetesクラスターでこれらのどのプロバイダーを使用していても有効です。
+{{< /note >}}
+
+
+<!-- steps -->
+
+## `nginx` Deploymentを作成してService経由で公開する
+
+Kubernetesのネットワークポリシーの仕組みを理解するために、まずは`nginx` Deploymentを作成することから始めましょう。
+
+```console
+kubectl create deployment nginx --image=nginx
+```
+```none
+deployment.apps/nginx created
+```
+
+`nginx`という名前のService経由でDeploymentを公開します。
+
+```console
+kubectl expose deployment nginx --port=80
+```
+
+```none
+service/nginx exposed
+```
+
+上記のコマンドを実行すると、nginx Podを持つDeploymentが作成され、そのDeploymentが`nginx`という名前のService経由で公開されます。`nginx`のPodおよびDeploymentは`default`名前空間の中にあります。
+
+```console
+kubectl get svc,pod
```

Comment:
```suggestion
kubectl get service,pod
```
?
I suggest this localization tweak because the abbreviation to “svc” is less obvious to people for whom English is a second language.

Reply:
nah this is ok

Reply:
Thanks for your suggestion. Yes, it's not so difficult to understand it. Personally I always use `svc` to reduce typing and others may do the same?

Reply:
Yeah. I usually use "svc" mainly.